### DISCUSSION TITLE
Interacting with Angular Three Examples

### DISCUSSION THREAD

**Discussion by IRobot1** [nartc/angular-three#107] [Ideas]
After going to all the effort of creating Angular Three, I find it funny that you're still resorting to a [2D method](https://angular-three.netlify.app/examples/home) of presenting the examples.  I think a 3D version of this would be cool.

One of my goals for work this year is to look for immersive alternatives for live, interactive data presentation in the area of industrial energy management.  It has to work within our existing Angular web application.  Hence the motivation to push Angular Three forward.

The most popular format for 3D so far is a mix between art gallery and conference booth format (think IKEA)
![image](https://user-images.githubusercontent.com/25032599/166310030-82c33054-ebac-4f05-8680-15b5249770fe.png)

Unreal engine content examples use a display room format for demonstrating specific game features.
![image](https://user-images.githubusercontent.com/25032599/166309918-3bc7bd79-ddc7-46ee-be9a-bef2c7d8f67f.png)

Moving around using the mouse and keyboard (or mini map) works well.  However, in VR, moving to your data doesn't work well.  

I think data has to surround the player and get their attention.  The player can then bring the data close to inspect and interact.

I plan to explore this and other alternative ideas over the coming year and will share my progress and failures.

**Comment by nartc** [comment]
I literally have 0 time to build all this stuffs by myself XD

**Comment by IRobot1** [comment]
Understood.  I appreciate the time you have given over the last month. Hopefully, I can help out in the examples area.

**Reply by nartc** [reply]
Love to receive contributions (you already done so much, appreciated it). But no pressure
